data/campaigns/Eastern_Invasion/achievements.cfg

```diff
@@ -1 +1,53 @@
 #textdomain wesnoth-ei
+
+#define ACHIEVEMENT ICON ID NAME DESC
+
+    [achievement]
+        icon,id,name,description={ICON},{ID},{NAME},{DESC}
+    [/achievement]
+#enddef
+#define HIDDEN ICON ID NAME DESC
+    [achievement]
+        icon,id,name,description={ICON},{ID},{NAME},{DESC}
+        hidden=yes
+    [/achievement]
+#enddef
+#define PATH
+data/campaigns/Eastern_Invasion/images/icons#enddef
+
+[achievement_group]
+    display_name=_"Eastern Invasion"
+    content_for=eastern_invasion
+
+    #
+    # many of these achievements are intended as content guides, teasing the player with things like "recruit the dunefolk" or "ally with the dwarves"
+    #    this helps players find content they might have missed
+    #
+    # but some achievements are VERY hard. Frenzied and Warmonger, for example.
+    # they are, at least, theoretically possible. But I would be very impressed if someone could achieve them on hard
+    #    (that's why it's called an "achieve"-ment, yeah?)
+    #
+
+    {ACHIEVEMENT {PATH}/escape-hatch.png      "ei_S01"    _"Scenario 1: Tactical Withdrawal"            _"Complete ‘<i>Eastern Invasion</i>’ with at least 10 surviving units."}
```

Comment:
As the scenario is the same name as the campaign, "Eastern Invasion" would need to be clarified. Renaming the scenario seems better.